### ISSUE TITLE
Exclude `No_Unfiltered_Uploads_Check` for PHP doc comment

### ISSUE THREAD

**Issue by ernilambar** [10up/plugin-check#1128]
Check: No_Unfiltered_Uploads_Check

This error is triggering for the match in PHP doc comment also.

```php
/**
 * Sample function.
 * 
 * Use like define('ALLOW_UNFILTERED_UPLOADS', true);
 */
function sample_function(){}
```

**Comment by davidperezgar** [comment]
I think the Plugin Check should flag `ALLOW_UNFILTERED_UPLOADS` even in comments because documentation serves as implementation guidance that developers often copy directly into production code.

By including references to disabling critical security features - even as examples - you're effectively promoting dangerous practices that bypass WordPress's file upload security restrictions. 

The WordPress.org directory should model security-first approaches, and code comments are not exempt from this standard since they influence developer behaviour and can easily become active code with a simple uncomment. 

I recommend removing or rephrasing the comment to avoid referencing this security constant, and instead describing the functionality conceptually or linking to official WordPress documentation if developers need advanced configuration information.
